kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ReplicasResource.java

```diff
@@ -114,12 +116,17 @@ public void getReplica(
         replicaManager.get()
             .getReplica(clusterId, topicName, partitionId, brokerId)
             .thenApply(replica -> replica.orElseThrow(NotFoundException::new))
-            .thenApply(replica -> GetReplicaResponse.create(toReplicaData(replica)));
+            .thenApply(replica -> GetReplicaResponse.create(toReplicaData(
+                replica,
+                crnFactory,
+                urlFactory)));
 
     AsyncResponses.asyncResume(asyncResponse, response);
   }
 
-  private ReplicaData toReplicaData(PartitionReplica replica) {
+  public static ReplicaData toReplicaData(PartitionReplica replica,
```

Comment:
@thomaskwscott looks like `TopicPartitionInfo.observer` information is not available in the Confluent Community version of Kafka lib. We show `Observer` info on the Topic overview page. 
We can either merge this implementation as is and add a new endpoint in ce-kafka-extension to fetch observer info or move this to change to `ce-kafka-extension` as is. Either way is fine with me.

Reply:
oh, I didn't realise we were after observer info. I'll move this to ce-kafka.

Reply:
@srpanwar-confluent just to confirm is it the /replicas or /replica-status endpoint we need a cluster level version of?